src/openrct2-ui/windows/Changelog.cpp

```diff
@@ -108,12 +108,7 @@ rct_window * window_changelog_open()
     int32_t screenHeight = context_get_height();
 
     window = window_create_centred(
-        screenWidth * 4 / 5,
-        screenHeight * 4 / 5,
-        &window_changelog_events,
-        WC_CHANGELOG,
-        WF_RESIZABLE
-    );
+        screenWidth * 4 / 5, screenHeight * 4 / 5, &window_changelog_events, WC_CHANGELOG, WF_RESIZABLE);
```

Comment:
It's kinda weird that the function arguments are on one line, but exclude the function name

Reply:
Yep, that's why I'm going to look into different penalties for breaking at different places.